modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,246 @@
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config]
+
+== Sources
+
+=== Data Fetcher
+
+`<graphql-router:data-fetcher>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Output MIME Type | String | The MIME type of the payload that this operation outputs. |  | 
+| Output Encoding | String | The encoding of the payload that this operation outputs. |  | 
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Primary Node Only | Boolean | Whether this source should only be executed on the primary node when running in cluster. |  | 
+| Streaming Strategy | 
+* xref:apikit-graphql-module-reference.adoc#repeatable-in-memory-stream[Repeatable In Memory Stream] 
+* xref:apikit-graphql-module-reference.adoc#repeatable-file-store-stream[Repeatable File Store Stream]
+* Non-Repeatable Stream | Configure if repeatable streams should be used and their behavior. |  | 
+| Redelivery Policy | xref:apikit-graphql-module-reference.adoc#redelivery-policy[Redelivery Policy] | Defines a policy for processing the redelivery of the same message. |  | 
+| Object Type Name | String | The object-type name |  | *x*
+| Field Name | String | The field name. |  | 
+| Body Content | Any | Content of the message (usually, the payload). | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+| *Attributes Type* | * xref:apikit-graphql-module-reference.adoc#data-fetcher-attributes[Data Fetcher Attributes] 
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config] 
+
+=== Data Loader
+
+`<graphql-router:data-loader>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Object Type Name | String | The object-type name. |  | *x*
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Primary Node Only | Boolean | Whether this source should only be executed on the primary node when running in cluster. |  | 
+| Redelivery Policy | xref:apikit-graphql-module-reference.adoc#redelivery-policy[Redelivery Policy] | Defines a policy for processing the redelivery of the same message. |  | 
+| Body Content | Any | Content of the message (usually, the payload). | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Array of Any
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config] 
+
+== Types
+
+=== Data Fetcher Attributes
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Alias | String | Alias of the queried field. |  | false
```

Comment:
line 21

Leave lowercase if literally "false".
Same comment on the following lines.

Reply:
As commented before, GraphQL is a case-sensitive language, so "false" is not the same as "False". So we should keep "false". Thanks!